man/add_analyzed_colvars.Rd

```diff
@@ -34,7 +34,7 @@ library(dplyr)
 ANL <- DM \%>\% mutate(value = rnorm(n()), pctdiff = runif(n()))
 
 # TODO: fix
-build_table(l, ANL)
+# build_table(l, ANL)
```

Comment:
I believe this is fixed, please (re)base your pr on the current state of master

Reply:
I did that, thanks